### PR TITLE
feat: Changes for OneSDK v2

### PR DESCRIPTION
**Pull request by janhalama**
This PR udates following documentation pages and guides with new SDK flow without local setup:

- [x] [Introduction](https://superface.ai/docs)
- [x] Describe super.json purpose in [super.json](https://superface.ai/docs/reference/superjson ) docs page
- [x] [Getting started](https://superface.ai/docs/getting-started)
- [x] [API keys](https://superface.ai/docs/guides/api-keys )
- [x] [Understand runtime integration ](https://superface.ai/docs/getting-started#understand-runtime-integration) paragraph should better explain the purpose of profile, caching of profile, and alternative approach with design time installation.

### CODE REVIEW

File: docs/getting-started.mdx [hunk 1]
```diff
@@ -30,6 +22,11 @@ Decide what capability you would like to use in your App. A capability is descri
 
 In this guide we will use the capability **List user repositories of a VCS** like GitHub or GitLab. Its ID is `vcs/user-repos`.
 
+:::note Capability catalog
+You can search for other capabilities in [Superface catalog](https://superface.ai/catalog).
```
Comment (janhalama) [inline docs/getting-started.mdx:26]: @SuperVrata  We are kind of inconsistent in terminology here. Once you click on the [Superface catalog](https://superface.ai/catalog) there is no mention of `capability`, instead `use case` is used everywhere.

Comment (janhalama) [reply docs/getting-started.mdx:26]: I propose to replace `capability` with `use case` in documentation as well.

Comment (tmladek) [reply docs/getting-started.mdx:26]: `capability` has been long deprecated AFAIK

Comment (janhalama) [reply docs/getting-started.mdx:26]: `capability` is used all over our docs. I will open separate PR to fix it.

File: docs/reference/one-sdk.mdx [hunk 2]
```diff
@@ -15,23 +15,16 @@ OneSDK is a universal API client for Node.js. This page describes OneSDK public
 
 ## Installation
 
-OneSDK is available as Node package under the name `@superfaceai/one-sdk`. For installation in a Node.js run:
+OneSDK is available as Node package under the name `@superfaceai/one-sdk`. For installation in a Node.js environment, run:
 
 ```shell
 npm install @superfaceai/one-sdk
+# Or, if you're using yarn
+yarn add @superfaceai/one-sdk
```
Comment (jnv) [inline docs/reference/one-sdk.mdx:23]: Personally I'd keep out the yarn out of any documentation. If you use it, you know how to use it, if you don't, it's annoying. Or if you insist on keeping yarn around, then you should also include pnpm. :wink:

File: docs/guides/create-new-capability.md [hunk 3]
```diff
@@ -34,8 +34,10 @@ Occasionally it makes sense to put multiple use-cases into a single profile file
 -->
 
 :::tip Scoped profiles
+
```
Comment (jnv) [inline docs/guides/create-new-capability.md:37]: Consider dropping these changes so it doesn't conflict with #95.

File: docs/guides/migration-v2.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# Migrating to SDK v2.0
+
+If you're reading this, then you're probably in the process of upgrading to OneSDK v2.0. Congratulations! This new version brings a lot of improvements. The most notable change is being able to **use OneSDK without `super.json`**, greatly simplifying usage if you're only just trying Superface out.
```
Comment (jnv) [inline docs/guides/migration-v2.md:3]: Ugh, that's really annoying and preachy tone.

File: docs/reference/one-sdk.mdx (hunk 2 above)
Comment (tmladek) [reply docs/reference/one-sdk.mdx:23]: Keeping this in line with all our READMEs, any reason to make the docs an outlier in this case?

File: docs/guides/create-new-capability.md (hunk 3 above)
Comment (tmladek) [reply docs/guides/create-new-capability.md:37]: And hotfix into `main`? I'd rather keep the changes, but you're right they don't belong in this PR

**Pull request merged**